google-cloud-pubsub/lib/google/cloud/pubsub/subscription.rb

```diff
@@ -308,7 +328,7 @@ def wait_for_messages max: 100
         #
         #   subscriber.start
         #
```

Comment:
I intentionally left examples off since I don't expect most users will want to change the default configuration.